### DISCUSSION TITLE
15 추론 및 pt저장 영상에서 colab

### DISCUSSION THREAD

**Discussion by na-seong** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#87] [2024년도 교육 자료 관련 질문] [answered]
15 추론 및 pt저장 영상에서 이 부분부터 진행이 안되는데 제가 방법을 몰라 도움 부탁드리고 싶습니다
![스크린샷 2024-07-16 23-46-35](https://github.com/user-attachments/assets/718f19aa-5bfe-4148-a311-37c8ed03bb40)

**Comment by HyeongKeunHong** [comment, marked as answer]
yolov8-seg.pt를 yolov8s-seg.pt로 수정하여 명령을 실행해보시기 바랍니다.

**Reply by na-seong** [reply]
아. 해결됐습니다 감사합니다!
